app/src/main/java/org/mifos/mobilebanking/models/User.kt

```diff
@@ -8,9 +9,9 @@ import java.util.ArrayList
  */
 
 data class User (
-
     var userId: Long = 0,
     var isAuthenticated: Boolean = false,
+    @SerializedName(value = "username")
```

Comment:
@ethan627hsu You approach is also correct, but you change the _userName_ to _username_.